pandas/tests/io/pytables/test_store.py

```diff
@@ -2689,16 +2687,12 @@ def test_select_dtypes(self, setup_path):
 
             expected = df[df.boolv == True].reindex(columns=["A", "boolv"])  # noqa
             for v in [True, "true", 1]:
-                result = store.select(
-                    "df", "boolv == {v!s}".format(v=v), columns=["A", "boolv"]
-                )
+                result = store.select("df", f"boolv == {v!s}", columns=["A", "boolv"])
```

Comment:
```suggestion
                result = store.select("df", f"boolv == {v}", columns=["A", "boolv"])
```

If you see any str representation constructs like this can just remove